### PR TITLE
feat(formatter): add `latexindent`

### PR DESCRIPTION
**Pull request by mathjiajia**

### CODE REVIEW

File: lua/guard/tools/formatter.lua [hunk 1]
```diff
@@ -64,9 +64,15 @@ M.isort = {
   stdin = true,
 }
 
+M.latexindent = {
```
Comment (glepnir) [inline lua/guard/tools/formatter.lua:67]:
```lua
ft('tex'):fmt({
  cmd = 'latexindent',
  fname = true,
  stdin = true,
})
```

Comment (xiaoshihou514) [reply lua/guard/tools/formatter.lua:67]: args = { "-" } stdin = true works for me

Comment (mathjiajia) [reply lua/guard/tools/formatter.lua:67]: yes.
IMO, there is no need to output a log file
That is why `args = { '-g', '/dev/null' },`

**Comment by glepnir** [comment]
hmm this not solved ? why need `/dev/null` ?

**Comment by xiaoshihou514** [comment]
> hmm this not solved ? why need `/dev/null` ?

-g specifies the path for log file

**Comment by xiaoshihou514** [comment]
> but this not work for windows.

! Right, staying simple then

**Comment by mathjiajia** [comment]
sorry, didn’t realise this issue on Windows 
Maybe just use `args = { ‘-‘ }`

**Pull request merged**